.ado/jobs/cli-init.yml

```diff
@@ -249,7 +249,9 @@ jobs:
             variables:
               - template: ../variables/vs2019.yml
 
-            pool: $(AgentPool.Medium)
+            pool:
```

Comment:
the interesting point is that variables can only be strings, but paramerters could be ojects...

At some point I considered making them parameters passed in but didn't push through, don't remember why..

Reply:
This change had me realize the ickiness of not being able to store a YAML object in variables 😢. Could hoist up as parameters, but this seems maybe a bit cleaner.